janus-gateway.spec

```
@@ -1,8 +1,8 @@
 %define janus_commit 922b3926e3b9ed2e50b6e6f36b6f018025dadf8b
```

Do we need to change it?

```suggestion
%define janus_commit cc204a56ee47de528b1b401e09ba58f710121a3d
```

---

Yes, absolutely right, we need to change it as reported by the doc 👍